verification/curator-service/api/test/model/data/automation.full.json

```diff
@@ -3,6 +3,7 @@
         "awsLambdaArn": "arn:aws:lambda:region:id:function:name"
     },
     "schedule": {
-        "awsRuleArn": "arn:aws:events:region:id:rule/name"
+        "awsRuleArn": "arn:aws:events:region:id:rule/name",
+        "awsScheduleExpression": "cron(0 10 * * ? *)"
```

Comment:
neither here nor there for this particular PR but wondering if we can actually use $ref so we don't have to re-specify sub-documents in these examples... not sure if they'll be dereferenced on import though... maybe something I'll try

i just wanna stay nice and DRY, you know?

Reply:
That'd be really neat -- let me know if you want a hand with that.